index.html

```diff
@@ -100,22 +100,22 @@ <h3>С 2004 года любим точность во всем:</h3>
               <ul class="jobs-list">
                 <li>
                     <a href="https://htmlacademy.ru/intensive/htmlcss">
-                        <img src="img/htmlacademy.jpg" width="220" height="165" alt="HTML academy">                    
+                        <img src="img/logo-1.png" width="199" height="68" alt="HTML academy">                    
```

Comment:
```suggestion
                        <img src="img/logo-html-academy.png" width="199" height="68" alt="HTML academy">                    
```
замени на читаемые названия, из которых понятно что это за картинки